crates/testenv/src/utils.rs

```diff
@@ -0,0 +1,90 @@
+use bdk_chain::bitcoin;
+
+#[allow(unused_macros)]
+#[macro_export]
+macro_rules! block_id {
+    ($height:expr, $hash:literal) => {{
+        bdk_chain::BlockId {
+            height: $height,
+            hash: bitcoin::hashes::Hash::hash($hash.as_bytes()),
+        }
+    }};
+}
+
+#[allow(unused_macros)]
+#[macro_export]
+macro_rules! h {
+    ($index:literal) => {{
+        bitcoin::hashes::Hash::hash($index.as_bytes())
+    }};
+}
```

Comment:
IIRC we discussed during the call that this could have a meaningful name, I  would use `hash` for instance.